src/ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusModal.jsx

```diff
@@ -1,34 +1,50 @@
 import React from 'react'
 import { translate } from 'cozy-ui/react'
+import Modal from 'cozy-ui/react/Modal'
 import Radio from 'cozy-ui/react/Radio'
+import Icon from 'cozy-ui/react/Icon'
+import { Title, Text } from 'cozy-ui/react/Text'
+import cx from 'classnames'
 import { List, Row } from 'components/List'
-import { PageModal } from 'components/PageModal'
+import iconReimbursement from 'assets/icons/icon-reimbursement-detailed.svg'
+import styles from 'ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusModal.styl'
+import { getReimbursementStatus, getLabel } from 'ducks/transactions/helpers'
 
 class _ReimbursementStatusModal extends React.PureComponent {
   render() {
-    const { currentStatus, onChange, t, ...rest } = this.props
+    const { transaction, onChange, t, className, ...rest } = this.props
     const choices = ['pending', 'reimbursed', 'no-reimbursement']
+    const status = getReimbursementStatus(transaction)
 
     return (
-      <PageModal {...rest}>
-        <form>
-          <List>
+      <Modal mobileFullscreen className={cx('u-pt-2', className)} {...rest}>
+        <header className="u-ta-center">
```

Comment:
Are you sure you do not want a `ModalHeader` here ?

Reply:
I could use a `ModalHeader`, but I would have to do custom styles around it because we are not 100% in the use case of this component.